src/views/darkroom.c

```diff
@@ -221,20 +221,19 @@ void expose(
   if(dev->image_status == DT_DEV_PIXELPIPE_DIRTY || dev->image_status == DT_DEV_PIXELPIPE_INVALID
      || dev->pipe->input_timestamp < dev->preview_pipe->input_timestamp)
   {
-    dt_control_queue_redraw();
-    dev->image_timeout_handle = g_timeout_add(DT_DARKROOM_PROCESS_TIMEOUT, G_SOURCE_FUNC(dt_dev_process_image), dev);
```

Comment:
This was done to avoid too many processing, the delay was making the UI more responsive IIRC (not tested this change). So I'm not sure this change is correct.

Note also that you have change rawspeed here, to avoid that make sure you do a:

`$ git submodule update`

Before committing changes on your side.

Reply:
Yes, the `g_timeout_add` is mandatory to avoid recomputing the pipe twice (or more) when zooming with the mouse scroll (otherwise, one computation is triggered at the beginning of the scroll event, and another at the end, and sometimes more in-between).